plugins/reference-assets/android/src/main/java/com/intuit/playerui/android/reference/assets/action/Action.kt

```diff
@@ -24,7 +23,7 @@ import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
 import kotlinx.serialization.Serializable
 
-class Action(assetContext: AssetContext) : ComposableAsset<Action.Data>(assetContext, Data.serializer()) {
+class Action(assetContext: AssetContext) : ComposableAsset<Action.Data>(assetContext, Data.serializer()), RenderableAsset.ViewportAsset {
```

Comment:
Hmm, I don't think `Action` shouldn't be a `ViewportAsset`. Or at least how that interface was used for Android View assets.